torch/_numpy/_dtypes_impl.py

```diff
@@ -0,0 +1,151 @@
+"""Dtypes/scalar type implementtaions with torch dtypes.
+
+Here `dtype` is always a torch.dtype, this module knows nothing about
+scalar types, wrapper dtypes or anything like that. PyTorch only.
+"""
+from collections import namedtuple
+
+import torch
+
+# defaults : mimic NumPy, allow user control
+DefaultDTypes = namedtuple(
```

Comment:
What's the public API for controlling this? Because I assume as a user I might want to change the default, right?

Reply:
We should expose `set_default_dtype` somehow. Probably as two flags in the dynamo config.

Reply:
```
In [1]: import torch_np as tnp

In [2]: tnp.set_default_dtype?
Signature: tnp.set_default_dtype(fp_dtype='numpy', int_dtype='numpy')
```

Reply:
Yeah, it's just that we wouldn't want people to have to deal with `torch._numpy` to do this. We would be better exposing this the same way we expose all the other options in dynamo, in `_dynamo.config`